pkg/virt-api/rest/subresource_test.go

```diff
@@ -2271,31 +2279,49 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			app.PauseVMIRequestHandler(request, response)
 
 			Expect(response.StatusCode()).To(Equal(http.StatusOK))
-
+			//In case of dry-run the request is not propagated to the handler
+			Expect(backend.ReceivedRequests()).To(handlerExpectation)
```

Comment:
```suggestion
			Expect(backend.ReceivedRequests()).To(matchExpectation)
```

It's just a rename but IMO that would read more nicely.